libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -6180,6 +6242,12 @@ bool GCS_MAVLINK::try_send_message(const enum ap_message id)
         CHECK_PAYLOAD_SIZE(SCALED_IMU3);
         send_scaled_imu(2, mavlink_msg_scaled_imu3_send);
         break;
+#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
+    case MSG_HIGHRES_IMU:
+        CHECK_PAYLOAD_SIZE(HIGHRES_IMU);
```

Comment:
if we end up sending more than 1 IMUs worth of data, this CHECK_PAYLOAD check will need to change

Reply:
We've learnt over time that keeping an offset into the thing you want to send and only sending one per loop is the way to go.